src/qt/walletcontroller.cpp

```diff
@@ -108,6 +108,13 @@ WalletModel* WalletController::getOrCreateWallet(std::unique_ptr<interfaces::Wal
     wallet_model->setParent(this);
     m_wallets.push_back(wallet_model);
 
+#if QT_VERSION < QT_VERSION_CHECK(5, 10, 0)
```

Comment:
There's a risk in implementing this twice. It's not strictly necessary because the pre-5.10 code will work fine on newer Qt. Might just as well remove the `#if` imo.

Reply:
Just wanted to make it easier for when we bump Qt. But will remove considering there are other places that would benefit the same approach.

Reply:
I understand, but I think it's better to change these (if desired) when bumping the *minimum supported* Qt version. Don't introduce conditional compilation when not necessary.

Reply:
Done :+1:

Reply:
Thanks!